website/src/css/custom.css

```diff
@@ -1,471 +1,497 @@
+
+    .theme-doc-version-banner {
+        font-size: 0.875rem;
+        border-top-left-radius: 0;
+        border-bottom-left-radius: 0;
+    }
+
+    .theme-doc-version-banner a {
+        text-decoration-color: currentColor;
+        text-decoration-thickness: revert !important;
+    }
+
+    .theme-doc-version-banner div.margin-top--md {
+        margin-top: 0 !important;
+    }
+
+    .theme-doc-version-badge {
+        font-size: 0.75rem;
+        border-radius: 0.25rem;
+        font-weight: normal;
+    }
+
+    div[class^="admonitionHeading"] {
+        text-transform: initial;
+    }
+
+    span[class^="admonitionIcon"] svg {
+        height: 1.2em;
+        width: 1.2em;
+    }
+
+    div.theme-admonition {
+        font-size: 0.875rem;
+        border-left-width: 2px;
+        border-top-left-radius: 0;
+        border-bottom-left-radius: 0;
+        /* border-radius: 0; */
+    }
+
+    .theme-admonition p {
+        margin: revert;
+    }
+
+    [data-theme="light"] img[src$="#gh-dark-mode-only"],
+    [data-theme="dark"] img[src$="#gh-light-mode-only"] {
+        display: none !important;
+    }
+
+    .pagination-nav__link {
+        padding: 0;
+        border: 0;
+    }
+
+    @media (prefers-reduced-motion: reduce) {
+        * {
+            animation-duration: 0 !important;
+            animation-iteration-count: 1 !important;
+            transition-duration: 0 !important;
+        }
+
+        html {
+            scroll-behavior: auto;
+        }
+    }
+
+    .navbar__inner,
+    div[class^="docRoot"] {
+        max-width: 100%;
+    }
+
+    .navbar__logo {
+        height: 1.75rem;
+    }
+
+    .markdown table.no-wrap-table {
+        display: block;
+    }
+
+    .no-wrap-table th,
+    .no-wrap-table td {
+        white-space: nowrap;
+    }
+
+    .no-wrap-table p {
+        margin: unset;
+    }
+
+    div[class^="tocCollapsible"] {
+        background-color: transparent;
+    }
+
+    div[class^="tocCollapsible"] > button {
+        padding-left: 0;
+        padding-top: 0;
+        padding-right: 0;
+    }
+
+    div[class^="tocCollapsibleContent"] > ul {
+        border-bottom: 1px solid var(--ifm-color-emphasis-300);
+    }
+
+}
+
+
+/*
+Increase specificity on purpose with the "impossible id selector", allows revert-layer to overrides everything
+See https://mayank.co/blog/revert-layer/
+*/
+.rdp-demo:not(#a#b) {
+    &,
+    * {
+        @layer docusaurus-infima {
+            all: revert-layer;
+        }
+
+        @layer docusaurus-website {
+            all: revert-layer;
+        }
+    }
+}
+
+.rdp-demo {
+    .rdp-root {
+        --rdp-accent-color: var(--ifm-color-primary);
+    }
 
-.markdown div[class^="codeBlockTitle_"] {
-  padding: 0.5rem 1rem;
-  font-size: 0.75rem;
+    html[data-theme="dark"] .rdp-root {
+        --rdp-accent-color: var(--ifm-color-primary);
+        --rdp-accent-background-color: #073845;
+        --rdp-range_end-color: black;
+        --rdp-range_start-color: black;
+    }
```

Comment:
I assume this should be put out of the `.rdp-demo` selector to work (probably it was added to support the shadow DOM).

Reply:
Thanks for your feedback, and glad to hear it seems to be working well!

Honestly I'm not sure exactly how your existing lib CSS rules work so I just tried to refactor things to make it look quite the same as it used to be, otherwise you can organize things differently if you think it's appropriate